docs/source/inversedesign.rst

```diff
@@ -4,8 +4,10 @@ Inverse design in photonics
 
 Adjoint optimization is a powerful tool that has gained significant attention in the photonics community. It leverages the adjoint method, a mathematical technique used to calculate gradients or derivatives of performance metrics with respect to design parameters. This method is particularly useful in photonics, where devices often have a large number of design parameters and complex performance metrics. This course is designed to provide a comprehensive understanding of the principles and applications of adjoint optimization in the field of photonics. 
 
-`Lecture 1: Introduction to Inverse Design In Photonics <https://www.flexcompute.com/fdtd101/Lecture-11-Introduction-to-Inverse-Design-In-Photonics/>`_
+`Lecture 1: Introduction to Inverse Design In Photonics <https://www.flexcompute.com/tidy3d/learning-center/inverse-design/Lecture-1-Introduction-to-Inverse-Design-In-Photonics/>`_
 
-`Lecture 2: Adjoint Method <https://www.flexcompute.com/fdtd101/Lecture-12-Inverse-Design-in-Photonics-Lecture-2-Adjoint-Method/>`_
+`Lecture 2: Adjoint Method <https://www.flexcompute.com/tidy3d/learning-center/inverse-design/Lecture-2-Inverse-Design-in-Photonics-Lecture-2-Adjoint-Method/>`_
+
+`Lecture 3: Adjoint Optimization <https://www.flexcompute.com/tidy3d/learning-center/inverse-design/Inverse-Design-in-Photonics-Lecture-3-Adjoint-Optimization/>`_
```

Comment:
lecture 4 is out now

https://www.flexcompute.com/tidy3d/learning-center/inverse-design/Inverse-Design-in-Photonics-Lecture-4-Fabrication-Constraints/

Reply:
Included!